src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java

```diff
@@ -27,15 +34,24 @@ protected Processor(
         this.documentProcessor = documentProcessor;
         this.envelopeProcessor = envelopeProcessor;
         this.errorWrapper = errorWrapper;
+
+        validator = Validation
+            .byProvider(HibernateValidator.class)
+            .configure()
+            .buildValidatorFactory()
+            .getValidator();
```

Comment:
Please keep it simple.
There's no need for a custom validation annotation which is tightly coupled to `ZipFileProcessor` and is used only once.
Just create a method.